express/server.js

```diff
@@ -111,16 +114,27 @@ app.get('/success', (req, res) => {
 });
 
 app.get('/products', async (req, res) => {
-  console.log("> /products")
   try {
+    // This /api call must happen before the DB.products() call or else it's a broken subtrace (if you do it after DB.Products())
+    await axios.get(RUBY_BACKEND + "/api")
+      .then(response => {
+        console.log("> response", response.data)
+        return
+      })
+      .catch(error => {
+        console.log(error);
+        Sentry.captureException(error)
+      });
+
     let transaction = Sentry.getCurrentHub()
       .getScope()
       .getTransaction();
     let span = transaction.startChild({ op: '/products.get_products', description: 'function' });
     const products = await DB.getProducts();
     span.finish();
-    transaction.finish();
```

Comment:
It turned out this line was never needed to begin with, because we're working off of the auto transaction that's already on Hub (it knows when to close itself)